### PR TITLE
feat(common): [components] add WToggleSwitch2

### PR DESCRIPTION
**Pull request by Tushar98644**
Part of #272 

Screenshot 2023-01-22 at 3 45 12 PM

Screenshot 2023-01-21 at 11 56 50 PM

### CODE REVIEW

**Comment by Tushar98644** [comment]
kindly review @meganindya

**Comment by Tushar98644** [comment]
> _ No description provided. _

Screenshot 2023-01-21 at 11 56 37 PM
Screenshot 2023-01-21 at 11 56 50 PM

**Comment by meganindya** [comment]
[Contributing Guidelines](https://github.com/sugarlabs/musicblocks-v4/blob/develop/docs/CONTRIBUTING.md)

And please follow the mockup

File: src/common/components/index.ts [hunk 1]
```diff
@@ -3,4 +3,6 @@ export { default as WTextButton } from './WTextButton';
 
 export { default as SImage } from './SImage';
 export { default as SImageVector } from './SImageVector';
+
+export { default as WToggleSwitch2 } from './WToggle-2';
```
Comment (meganindya) [inline src/common/components/index.ts]: insert between lines 1 and 2

File: src/common/components/WToggleSwitch/index.scss [hunk 2]
```diff
@@ -41,4 +41,4 @@
       }
     }
   }
-}
+}
```
Comment (meganindya) [inline src/common/components/WToggleSwitch/index.scss:44]: this file shouldn't be a part

File: src/common/components/WToggle-2/index.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Active.args = {
+  active: true,
+  handlerClick: linkTo('Common/WToggleSwitch2', 'InActive'),
+};
+
+export const Inactive = Template.bind({});
+Inactive.args = {
+  active: false,
+  handlerClick: linkTo('Common/WToggleSwitch2', 'Active'),
+};
```
Comment (meganindya) [inline src/common/components/WToggle-2/index.stories.tsx]: this doesn't align with storybook v7. I updated storybook from v6.5.15 to v7.0.0-beta.31

File: src/common/components/WToggle-2/index.scss [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+      background: $mb-accent;
+
+      .w-toggle-handle-1 {
+        left: calc(100% - 5px);
+        transform: translateX(-100%);
+
+      }
+    }
+  }
+}
```
Comment (meganindya) [inline src/common/components/WToggle-2/index.scss]: the dimensions don't match the mockup. Also please use color variables for white and grey.

File: src/common/components/WToggle-2/index.scss [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+    }
+  }
+
+  &.w-toggle-active-1 {
+    .w-toggle-track-1 {
+      background: $mb-accent;
+
+      .w-toggle-handle-1 {
+        left: calc(100% - 5px);
+        transform: translateX(-100%);
```
Comment (meganindya) [inline src/common/components/WToggle-2/index.scss]: this might result in animation artefacts.

**Review by meganindya — CHANGES_REQUESTED** [review]
I'd appreciate sticking to the naming convention specified in the issue. `WToggleSwitchRounded` for now.

**Comment by meganindya** [comment]
Please rebase on top of develop and squash down everything to one commit. You shouldn't merge default branch into a feature branch; always rebase.

**Pull request closed without merging**